convex/init.ts

```diff
@@ -1,5 +1,5 @@
 import { api } from "./_generated/api";
-import { internalMutation } from "./_generated/server";
+import { internalMutation, type MutationCtx } from "./_generated/server";
```

Comment:
oh cool, I don't think I've noticed that before. I just blindly import things. TIL